android/src/app/organicmaps/MwmActivity.java

```diff
@@ -855,8 +856,6 @@ protected void onSaveInstanceState(@NonNull Bundle outState)
         fragment.saveRoutingPanelState(outState);
     }
 
-    mNavigationController.onActivitySaveInstanceState(this, outState);
```

Comment:
What happens if the app is killed by the system? AFAIR the current implementation restored and continued navigation.

Reply:
@Jean-BaptisteC can you please test this case on the current master and on this branch?

Reply:
mNavigationController.onActivitySaveInstanceState() here was no-op, navigation state is saved in a different place. I tested that navigation is restarted when the app was killed.